src/pevm.rs

```diff
@@ -252,13 +252,23 @@ impl Pevm {
                     None
                 };
 
-                // TODO: Assert that the evaluated nonce matches the tx's.
                 for (tx_idx, memory_entry) in write_history.iter() {
                     match memory_entry {
                         MemoryEntry::Data(_, MemoryValue::Basic(info)) => {
+                            // TODO: these debug assertions can later be converted to errors
```

Comment:
We have two issues tracking it!
- https://github.com/risechain/pevm/issues/8
- https://github.com/risechain/pevm/issues/341

We'll have a round of converting these panics into proper error types. Another round to disable less serious checks for people who "own" the input data and want the most competitive speed.

So I think we don't need to add this comment here, otherwise for completeness we should add to all others too!